docusaurus/i18n/en/code.json

```diff
@@ -0,0 +1,6 @@
+{
+  "theme.common.editThisPage": {
+    "message": "Improve this page",
+    "description": "The link label to edit the current page"
```

Comment:
The `description` here is the default one from Docusaurus.

Is not displayed in the documentation.
It needs to be in this format, [according to the `i18` guides](https://docusaurus.io/docs/i18n/introduction#json-files), by helping translation tools to have more context.